src/router/storeSync/storeSync.config.js

```diff
@@ -104,6 +105,15 @@ const storeSyncConfig = [
     new CrossHairParamConfig(),
     new CompareSliderParamConfig(),
     new LayerParamConfig(),
+    new SimpleUrlParamConfig(
+        'featureInfo',
+        'setFeatureInfoPosition',
+        'setFeatureInfoPosition',
+        (store) => store.state.ui.featureInfoPosition,
+        false,
+        String,
+        FeatureInfoPositions.DEFAULT
```

Comment:
I suggest to set `NONE` as default, so it make easier to create a new permalink without info, but we can quickly discuss this in refinement.

Reply:
setting it to default would also simplify the legacy parameter, as it was the default behavior of 'showTooltip' . We can discuss this quickly tomorrow and see if anyone has another idea